rancher/rancher-compose/rancher-services/index.md

```diff
@@ -0,0 +1,186 @@
+Alias/DNS Service | rancher/dns-service
+
+### Load Balancer
+
+A load balancer can be scheduled like any other service. Read more about [scheduling]({{site.baseurl}}/rancher/rancher-compose/scheduling/) services and load balancers using `rancher-compose`.
+
+Load balancing using the basic use case of load balancers is simply adding ports and linking services. Any traffic directed to a source port will be sent to the private port of the linked services.
+
+We go into more detail on [advanced routing options]({{site.baseurl}}/rancher/rancher-ui/applications/stacks/adding-balancers/#advanced-routing-optons) for load balancers in our UI section, but anything that we can create in Rancher can be created using `rancher-compose`.
+
+Advanced routing options use `labels` in the `docker-compose.yml` file. Here's the basic syntax for applying the label. You would only use this label if you were going to do some advanced routing rules.
+
+Label Key | Label Value
+---| ---
+`io.rancher.loadbalancer.target.SERVICE_NAME` | `REQUEST_HOST:SOURCE_PORT/REQUEST_PATH=TARGET_PORT`
+
+> **Note:** If you use a source port with the advanced routing options, the source port must also be listed in `ports` section of the `docker-compose.yml`
+
+Here is the syntax for all combinations using request host, source port, path and target port. 
+
+Request Host | Source Port | Path | Target Port | Label Value
+---|---|---|---| ---
+example.com | 80 | path | 81 | `example.com:80/path=81`
+example.com | 80 | path | | `example.com:80/path`
+example.com | 80||81 | `example.com:80=81`
+example.com |80 | | | `example.com:80`
+example.com | | path | 81 | `example.com/path=81`
+example.com | | path | | `example.com/path`
+example.com| | | 81 | `example.com=81`
+example.com | | | | `example.com`
+ | 80| path | 81 | `80/path=81`
+ | 80 |path | | `80/path` 
+ |80 | | 81 | `80=81`
+ | | path | 81| `/path=81`
+ | | path | | `/path`
+ | | | 81 | `81` *See Note 
+
+<br>
+**Note:** It is assumed that if you have only a port in the label, then the port is for the target port of the service. 
+
+**Load Balancer Sample configuration** `docker-compose.yml`
+
+```yaml
+lb:
+  image: rancher/load-balancer-service
+  ports:
+  # Listen on public port 80 and direct traffic to private port 80 of the service
+  - 80
+  # Listen on public port 82 and by default forward traffic to private port 81 using HTTP
+  - 82:81
+  # Listen on public port 9999 using TCP and not HTTP
+  - 9999/tcp
+  links: 
+  # Any service that is a target will be listed as a link
+  - web1:web1
+  - web2:web2
+  - web3:web3
+  labels:
+    # Put load balancer containers on hosts with label lb=true
+    - "io.rancher.scheduler.affinity:host_label=lb=true"
+    # Requests to http://app.example.com/foo:80 should be routed to web1 over port 8000
+    - "io.rancher.loadbalancer.target.web1=app.example.com:80/foo=8000"
+    # Requests to http://app.example.com should be routed to web2 over port 8000
```

Comment:
1) We should mention that its possible to configure multiple hostname routing rules for the same service, in this case they should be separated by coma (see an example below)

2) Rancher-compose supports 2 ways for defining the labels: 

- as a list of strings of following format:

"io.rancher.loadbalancer.target.web1=app.example.com:80/foo=8000, app.example.com:80/bar=800"

- as a key/value pair:

io.rancher.loadbalancer.target.web1: app.example.com:80/foo=8000, app.example.com:80/bar=800


I believe Darren wants the doc to reflect the second way; but can you please double check with Son that this format works for affinity rules. I've checked for hostname routing rles